1_preprocessing_data/README.md

```diff
@@ -2,13 +2,37 @@
 
 In this module, we present our pipeline for preprocessing the NF1 Schwann Cell data.
 
-## Illumination Correction
+## Illumination Correction (IC)
 
 To correct for illumination issues within the NF1 data, we use the BaSiC method that was established in an article by [Peng et al.](https://doi.org/10.1038/ncomms14836).
 We specifically use the Python implementation of this method, called [PyBaSiC](https://github.com/peng-lab/BaSiCPy).
 
 Illumination correction is an important step in cell image analysis pipelines as it helps with downstream processes like segmentation (more accuracy in segmenting shape and identifying objects to segment) and feature extraction (accurate measurements in intensity, texture, etc.).
 
+Visualizing if images need IC can be simple or difficult depending on the dataset. 
+In the NF1 dataset, identifying illumination issues is impossible to capture with the human eye.
+
+To visualize the illumination issues, we use Fiji to increase the maximum limits of the display range to brighten the images using the `B&C` tool (Figure 1).
+
+![NF1_IC_Demonstration.png](example_figs/NF1_IC_Demonstration.png)
+> Figure 1. Use of Fiji to Determine Illumination Issues. This figure shows how increasing the maximum limit for the display range reveals illumination issues (e.g. vignetting like in the DAPI channel). The increasing of the upper limits for display starts to show where there is brighter areas in the image that was hidden when the upper limit was lower.
```

Comment:
```suggestion
> Figure 1. Use of Fiji to determine illumination issues. This figure shows how increasing the maximum limit for the display range reveals illumination issues (e.g. vignetting like in the DAPI channel). Increasing upper limits shows brighter areas in the image that were hidden previously.
```

edited for lower case and understanding. Please check me!

Reply:
That is a much more concise way of putting it! Fixed!